ENIGMAsystem/SHELL/Platforms/General/POSIX/POSIXfilemanip.cpp

```diff
@@ -83,9 +83,7 @@ string file_find_next() {
 
   const int not_attrib = ~fff_attrib;
 
-  if (r == "." or r == ".."                                                      // Don't return ./ and
-      or ((r[0] == '.' or r[r.length() - 1] == '~') and not_attrib & fa_hidden)  // Filter hidden files
-  )
+  if ((r[0] == '.' or r[r.length() - 1] == '~') and not_attrib & fa_hidden)  // Filter hidden files
```

Comment:
Why did you change this?

Reply:
See screenshot I posted fucktard. I proved you wrong about like 3 things today. Believe it or not, there are cases where that .. and . are very useful. One use would be to make your own custom gui alternative for get_open_filename skinned with your own sprites rather than opening an external window with a native look. This would allow adding a button that links to the parent folder if one exists.

Reply:
I will say this here, as well. Your attitude on this matter is NOT acceptable. Just because Windows as Game Maker share a behavior does not mean that ENIGMA should have it. We seek to be useful as a game development platform. Not to simulate glitches and nuances in 5-year-old, 10-year-old, or 50-year-old APIs.

This isn't some game where you say "I proved you wrong!!" and now you win. Regardless of who has learned what about these APIs, today, it is NOT RELEVANT to this discussion how they behave. We should be looking at what it means to build a good toolkit.

If a project needs . and .. for some reason, it's easy to tack them on. They are unreliable in these systems, and are generally hard to remove. It should not be the default behavior. Your PR yesterday showed a great example of the usual case; a user's first order of business being to continue iteration as soon as . or .. are encountered.

I would not be strictly opposed to adding a flag to include them, but I don't value the behavior and would sooner get rid of it entirely.